Modules/_ssl_data.h

```diff
@@ -134,11 +134,6 @@ static struct py_ssl_error_code error_codes[] = {
   #else
     {"SHORT_HEADER", ERR_LIB_PEM, 112},
   #endif
-  #ifdef PEM_R_UNSUPPORTED_CIPHER
-    {"UNSUPPORTED_CIPHER", ERR_LIB_PEM, PEM_R_UNSUPPORTED_CIPHER},
-  #else
-    {"UNSUPPORTED_CIPHER", ERR_LIB_PEM, 113},
-  #endif
```

Comment:
Hmmm this error was accurate..., pem unsupported cipher value is 113 too.
Is this removed because we can't detect the source of error and 113 (ssl source) is invalid configuration?

Contrast this change with the change for error 112 which is kept at pem source and also updated at ssl source.